src/proxy/http/remap/PluginFactory.cc

```diff
@@ -175,6 +183,20 @@ PluginFactory::getRemapPlugin(const fs::path &configPath, int argc, char **argv,
       runtimePath /= _runtimeDir;
       runtimePath /= effectivePath.relative_path();
 
+      // Special case for Cripts
+      if (!runtimePath.string().ends_with(".so")) {
+        if (_compilerPath.empty()) {
+          error.assign("compiler path not set for compiling plugins");
+          return nullptr;
+        }
+
+        // Add .so to the source file, so e.g. example.cc.so. ToDo: libswoc doesn't allow appending to the extension.
+        std::string newPath = runtimePath.string();
```

Comment:
Oh, I suppose we will compile it to `example.cc.so`. It shouldn't be too hard to remove the extension after the last `.` right?